mountpoint-s3/src/fs.rs

```diff
@@ -334,6 +335,26 @@ impl Default for CacheConfig {
     }
 }
 
+impl From<TimeToLive> for CacheConfig {
+    fn from(value: TimeToLive) -> Self {
+        match value {
+            TimeToLive::Minimal => Default::default(),
+            TimeToLive::Indefinite => Self {
+                serve_lookup_from_cache: true,
+                file_ttl: TimeToLive::INDEFINITE_DURATION,
+                dir_ttl: TimeToLive::INDEFINITE_DURATION,
+                ..Default::default()
+            },
+            TimeToLive::Custom(ttl) => Self {
+                serve_lookup_from_cache: true,
+                file_ttl: ttl,
+                dir_ttl: ttl,
+                ..Default::default()
+            },
+        }
+    }
+}
+
```

Comment:
Even though API breakage isn't as much a concern in this crate, I probably wouldn't do this as a `From` impl, as we'll just end up having to undo it in the future if we ever add more cache configs.